apps/full-site-editing/package.json

```diff
@@ -41,7 +41,7 @@
 		"clean": "npx rimraf dist full-site-editing-plugin/*/dist",
 		"prebuild": "npm run clean",
 		"predev": "npm run clean",
-		"sync:homepage-posts": "./bin/sync-homepage-posts.sh"
+		"sync:blog-posts-block": "./bin/sync-blog-posts-block.sh"
```

Comment:
why not instead of setting a name which could change again (🤞) define a name not related to the FSE plugin name? 

`sync:newspack-articles`, for instance. In the end, it's what it does.

Reply:
Newspack-articles doesn't seem to be used anywhere else. I think `blog-post-block` makes sense, given that we're syncing that block.